db/migrate/20150826203108_create_gem_ownership_transfers.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGemOwnershipTransfers < ActiveRecord::Migration
+  def change
+    create_table :gem_ownership_transfers do |t|
+      t.integer :old_user_id, null: false, index: true
+      t.integer :new_user_id, null: false, index: true
+      t.integer :ruby_gem_id, null: false, index: true
+
+      t.timestamps null: false
+    end
+  end
```

Comment:
You make add this to make all columns [foreign keys](http://apidock.com/rails/ActiveRecord/ConnectionAdapters/TableDefinition/foreign_key) so that Rails also add indexes which is probably a good idea:
```ruby
add_foreign_key :gem_ownership_transfers, :users, column: :old_user_id
add_foreign_key :gem_ownership_transfers, :users, column: :new_user_id
```